### DISCUSSION TITLE
Boss installation ?

### DISCUSSION THREAD

**Discussion by dcabale** [hazzelnuts/ntfy-for-delphi#37] [General]
Hi there,
what is this "Boss installation" you are [referring to](https://github.com/hazzelnuts/ntfy-for-delphi#%EF%B8%8F-boss-installation) and for what use ?
Thanks

**Comment by p-samuel** [comment]
@dcabale, in case you are not familiar, boss is a Delphi dependency manager created by [hashload](https://github.com/HashLoad) community. It works with semantic versioning so that you can manage your project packages in a way similar to npm. There are other Delphi packages manager as well but [boss](https://github.com/HashLoad/boss) is quite simple and neat. Take a look on their page for more information. Summarizing, you just need to add the binary in some location on your computer and then inform it on the PATH variable. After this procedure boss will be available on your command terminal so you can start using on your Delphi projects.
